src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -13703,6 +13703,10 @@ emitter::insFormat emitter::getMemoryOperation(instrDesc* id)
 
 #if defined(DEBUG) || defined(LATE_DISASM)
 
+#ifndef add
+#define add(a, b) ((a) + (b))
```

Comment:
Why is this needed? Seems a pretty weird place to put this.

Reply:
Some places I use max(a,b) to compute the instruction Latencies and now I see that that some instructions should instead compute the latency using  add(a,b).

I could rewrite these using a + b, if you feel strongly.

Reply:
In my opinion, we shouldn't create macros for simple arithmetic operations.

Reply:
OK I will change these